<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtocolDlg</class>
 <widget class="QDialog" name="ProtocolDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="combo_manu">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_manu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>制造商：</string>
   </property>
   <property name="buddy">
    <cstring>combo_manu</cstring>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_protocol">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_protocol">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>39</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>系列：</string>
   </property>
   <property name="buddy">
    <cstring>combo_protocol</cstring>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>321</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>通讯设置</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>站号：</string>
    </property>
    <property name="buddy">
     <cstring>spin_stationIndex</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_stationIndex">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>超时时间：</string>
    </property>
    <property name="buddy">
     <cstring>spin_time</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_time">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="maximum">
     <number>300000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>毫秒</string>
    </property>
    <property name="buddy">
     <cstring>spin_stationIndex</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最大读写长度：</string>
    </property>
    <property name="buddy">
     <cstring>spin_length</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_length">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>256</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>字节</string>
    </property>
    <property name="buddy">
     <cstring>spin_stationIndex</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>重试次数：</string>
    </property>
    <property name="buddy">
     <cstring>spin_retry</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_retry">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>帧间隔时间：</string>
    </property>
    <property name="buddy">
     <cstring>spin_interval</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_interval">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>30000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>毫秒</string>
    </property>
    <property name="buddy">
     <cstring>spin_stationIndex</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>次</string>
    </property>
    <property name="buddy">
     <cstring>spin_stationIndex</cstring>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_cycleTime">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60000</number>
    </property>
    <property name="value">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>最小通信周期：</string>
    </property>
    <property name="buddy">
     <cstring>spin_interval</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>毫秒</string>
    </property>
    <property name="buddy">
     <cstring>spin_stationIndex</cstring>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_UdpcheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_default">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>恢复默认值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IP">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>204</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_port">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>链接名称中不能包含'#'号</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhPreferNumbers</set>
    </property>
    <property name="text">
     <string>12345</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_port">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>234</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>端口号：</string>
    </property>
   </widget>
   <widget class="IpAddressWidget" name="edit_IP" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>192</y>
      <width>181</width>
      <height>36</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_manu_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
   <property name="buddy">
    <cstring>edit_name</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>99</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>400</y>
     <width>239</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_accept">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_help">
      <property name="text">
       <string>帮助</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IpAddressWidget</class>
   <extends>QWidget</extends>
   <header location="global">frame\akipedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
